app/views/course/forum/topics/_topic.html.slim

```diff
@@ -0,0 +1,19 @@
+= content_tag_for(:tr, topic) do
+  td
+  td
+    h4
+      = link_to(topic.title, course_forum_topic_path(@course, @forum, topic))
+    div.started-by
+      = t('.started_by')
+      = topic.creator.name
+  td = topic.votes_count
+  td = topic.post_count
+  td = topic.view_count
+  td.latest-post
+    - last_post = topic.posts[0]
+    - if last_post
+      = t('.last_post')
```

Comment:
hmm... I mean which part should be treated as a parameter to pass in the 't'? Link_to?